ghga_connector/core/decorators.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
```

Comment:
Maybe we rather call this module "retry" or "utils" (if we want to be more generic). "decorators" doesn't tell much about the contained functionality.